codice/ddf
# DDF-04533 Migrate list-create to React

#### What does this PR do?
Migrates list-create to React
Conditionally render editor-footer in list-editor based on a variable, rather than hiding via inherited css/less from list-create. 

#### Who is reviewing it? 
@Schachte 
@middlets719 
@gordocanchola 
@adimka


#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@bdeining

#### How should this be tested?

Open a workspace
Navigate to 'Lists'
Create a new list
Verify that the created list shows up. 
Navigate to 'Search'
Perform a search that returns some results.
On any result click the '+' icon
Click 'Create New List' and create a list
Verify that the list was successfully created and shows up in the Lists tab

#### Any background context you want to provide?
#### What are the relevant tickets?
For GH Issues:
Fixes: #4533 

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
